package.json

```diff
@@ -44,19 +44,23 @@
     "cheerio": "0.22.0",
     "commander": "2.16.0",
     "express": "4.16.3",
+    "file-type": "8.0.0",
```

Comment:
Just before you merge, it's probably worth checking these are up to date, as I'm pretty sure we only get PRs for updates after it's merged in.

Reply:
Yep, updated some. Good call.